internal/command/checks/command.go

```diff
@@ -0,0 +1,90 @@
+package checks
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"golang.stackrox.io/kube-linter/internal/builtinchecks"
+	"golang.stackrox.io/kube-linter/internal/check"
+	"golang.stackrox.io/kube-linter/internal/command/common"
+	"golang.stackrox.io/kube-linter/internal/defaultchecks"
+	"golang.stackrox.io/kube-linter/internal/ternary"
+)
+
+var (
+	dashes = func() string {
+		var sb strings.Builder
+		for i := 0; i < 30; i++ {
+			sb.WriteRune('-')
+		}
+		return sb.String()
+	}()
+
+	formatsToRenderFuncs = map[string]func([]check.Check, io.Writer){
```

Comment:
Possibly irrelevant (see below), but the brace following the closing parens without space in between looks weird - doesn't the linter flag this?

Reply:
That brace is actually consistent with `go fmt` since it applies to the `map`, and not the `func` (although I agree it pattern matches with braces around a `func` and looks weird).